aspnetcore/mvc/views/tag-helpers/builtin/resources/AnchorTagHelper.md

```diff
@@ -8,41 +8,33 @@ By [Peter Kellner](http://peterkellner.net)
 
 The ```AnchorTagHelper``` enhances the html anchor (`<a ... ></a>`) tag. A new set of attributes are defined that work with the anchor tag. The link generated (on the href tag) is based on a combination of these new attributes that work together to form the final redirect URL. That URL can include an optional protocol such as https.
 
-For reference, the following ASP.NET ```startup.cs``` and ```SpeakerController.cs``` are defined as you would expect in a default Visual Studio .Net Core Web Project.
+For reference, the following ASP.NET ```SpeakerController.cs``` is defined as you would expect in a default Visual Studio .Net Core Web Project.
 
-![](../_static/ProjectControllers.png)
 
-**Startup.cs**
-```
-// This method gets called by the runtime. 
-// Use this method to configure the 
-// HTTP request pipeline.
-public void Configure(IApplicationBuilder app, 
-    IHostingEnvironment env, 
-    ILoggerFactory loggerFactory)
-{
-    app.UseMvcWithDefaultRoute();
-    //...
-}  
-```
 
 <br/>
 **SpeakerController.cs** 
 
 [!code-csharp[SpeakerController](../sample/TagHelpersBuiltInAspNetCore/src/TagHelpersBuiltInAspNetCore/Controllers/SpeakerController.cs)]
 
-<br/>
-These attributes are defined as follows:
 
-## asp-controller
+## The attributes are defined as follows
+
+- - -
+
+### asp-controller
 
 ```asp-controller``` is used to associate which controller will be used to generate the final URL. The only valid choices are controllers that exist in the current project. In our case, to get to a list of all speakers or speaker details you would specify ```asp-controller="Speaker"```. If you only specify ```asp-controller``` and no ```asp-action``` the URL will generate without an error but will likely not be what you expect.
```

Comment:
Drop the first prepositional phrase and move the comma ... "To get to a list of all speakers or speaker details, you would ..."

> If you only specify ```asp-controller``` and no ```asp-action``` the URL will generate without an error but will likely not be what you expect.

I find the construction a bit awkward ... "... will generate without ..." ... not a common way to phrase this thought in spoken English. Anyway ... the comma ... at least set the phrase off with a comma: "If you only specify `asp-controller` and no `asp-action`, the URL will ..."